src/github.com/getlantern/lantern-mobile/app/src/main/java/org/getlantern/lantern/activity/LanternMainActivity.java

```diff
@@ -137,7 +143,6 @@ protected void onDestroy() {
             if (mReceiver != null) {
                 unregisterReceiver(mReceiver);
             }
-            stopLantern();
```

Comment:
We no longer call ``stopLantern()`` when the main activity finishes, to leave Lantern running in the background, to also resolve https://github.com/getlantern/lantern/issues/3711